src/coreclr/src/nativeaot/System.Private.CoreLib/src/System/Runtime/TypeLoaderExports.cs

```diff
@@ -392,48 +392,34 @@ private static Entry[] ResizeCacheForNewEntryAsNecessary()
     [ReflectionBlocked]
     public delegate IntPtr RuntimeObjectFactory(IntPtr context, IntPtr signature, object contextObject, ref IntPtr auxResult);
 
-    [System.Runtime.InteropServices.McgIntrinsicsAttribute]
-    internal class RawCalliHelper
+    internal unsafe class RawCalliHelper
```

Comment:
We should be able to replace `RawCalliHelper` with CallConv attribute on the function itself to clean this up further.

Reply:
Oh, yes. The C# compiler doesn't care about the names of calling conventions - we can make a new CallConv type and the C# compiler will just respect that. Neat!